src/loader.js

```diff
@@ -362,7 +362,7 @@ function setupModuleLoader(window) {
           component: function(name, options) {
             function factory($injector) {
               function makeInjectable(fn) {
-                if (angular.isFunction(fn)) {
+                if (isFunction(fn) || isArray(fn)) {
```

Comment:
It is also accessing `minErr` near the top of the file. It definitely relies on having access to some stuff. (Whether it _should_ is another story...)